static/sharing.ts

```diff
@@ -120,7 +120,7 @@ export class Sharing {
         const stringifiedConfig = JSON.stringify(config);
         if (stringifiedConfig !== this.lastState) {
             if (this.lastState != null && window.location.pathname !== window.httpRoot) {
-                window.history.replaceState(null, null, window.httpRoot);
+                window.history.replaceState(null, '', window.httpRoot);
```

Comment:
Second argument in replaceState/pushState is unused, but of type string, not null. Changed the value accordignly